schema-source/RSxxGlzSys.schema.yaml

```diff
@@ -0,0 +1,261 @@
+      Constraints: ">=0"
+      Required: if fill=OTHER
+
+FillOptions:
+  Object Type: "Enumeration"
+  Enumerators:
+    AIR:  
+      Display Text: "Air"
+    ARGON:
+      Display Text: "Argon"
+    KRYPTON:
+      Display Text: "Krypton"
+    XENON:
+      Display Text: "Xenon"
+    VACUUM:  
+      Display Text: "Vacuum"
+    OTHER:
+      Display Text: "Other"
+
+FutureRSxxAttach:
+  Object Type: "Data Group"
+  Data Elements:
+    name:
+      Description: "name"
+      Data Type: "String"
+      Notes: "This is just a placeholder. This is referencing an RS that has not yet been developed but will represent blinds, shades, etc."
+
+ColorTransmittance:
+  Object Type: "Data Group"
+  Data Elements:
+    color_rendering_indices:
+      Description: "List of color rendering indices"
+      Data Type: "[{ColorRenderingIndex}]"
+      Notes: "The Color Rendering Index (CRI) indicates how well the transmitted radiation matches daylight regarding the rendering of colors. `Ra` considers one transmission of the building envelope. `Raout-in` considers two transmissions of the building envelope."
+    cielab_colors:
+      Description: "List of CIELAB colors"
+      Data Type: "[{CielabColors}]"
+      Notes: "The three color coordinates L* (lStar), a* (aStar) and b* (bStar) defined by the International Commission on Illumination (CIE) define the color which is reflected by a component or transmitted through a component."
+
+ColorReflectance:
+  Object Type: "Data Group"
+  Data Elements:
+    cielab_colors:
+      Description: "List of CIELAB colors"
+      Data Type: "[{CielabColors}]"
+      Notes: "The three color coordinates L* (lStar), a* (aStar) and b* (bStar) defined by the International Commission on Illumination (CIE) define the color which is reflected by a component or transmitted through a component."
+
+ColorRenderingIndex:
+  Object Type: "Data Group"
+  Data Elements:
+    ra: 
+      Description: "Color Rendering Index CRI Ra value or Ra-inout."
+      Data Type: "Numeric"
+      Required: True
+      Constraints: ">=0, <=100"
+      Notes: "If is_ra_in_out is False, the Color Rendering Index as it is defined by EN 410 based on CIE 13. 3-1995, Method of Measuring and Specifying Colour RenderingProperties of Light Sources, 1995, ISBN: 9783900734572. It indicates how well colors are rendered after one transmission of the building envelope. For example, how the color of an object inside a building is perceived from within the building.  If is_ra_in_out is True, The Color Rendering Index Raout-in is defined by T.E. Kuhn, H.R. Wilson, J. Hanek, M. Santamouris, Ra out-in Color rendering of objects in a daylit room viewed from outdoors, Energy and Buildings 118 (2016) 93�98, https://doi.org/10.1016/j.enbuild.2016.02.019. It indicates how well colors are rendered after two transmissions of the building envelope. For example, how the color of an object inside the building is percived from the outside."
+    is_ra_in_out:
```

Comment:
Do you want to allow only one Color Rendering Index? Why not allow both, the regular for an observer inside and additional for an observer outside of the building?

Reply:
The data group "ColorTransmittance" allows for an array of ColorRenderingIndex so more than one is allowed.